PokeStar/PokeStar/Modules/RaidCommands.cs

```diff
@@ -288,59 +339,63 @@ private static string BuildRaidTitle(int tier)
       {
          StringBuilder sb = new StringBuilder();
          sb.Append("Raid ");
-
          string raidSymbol = Emote.Parse(Environment.GetEnvironmentVariable("RAID_EMOTE")).ToString();
-
          for (int i = 0; i < tier; i++)
             sb.Append(raidSymbol);
-
          return sb.ToString();
       }
 
-      private static string BuildPingList(List<SocketGuildUser> players, string loc)
+      private static string BuildPingList(List<SocketGuildUser> players, string location, int groupNumber)
       {
          StringBuilder sb = new StringBuilder();
-
          foreach (SocketGuildUser player in players)
             sb.Append($"{player.Mention} ");
-         sb.Append($"Everyone is ready at {loc}");
+         sb.Append($"Everyone in group {groupNumber + 1} is ready to jump at {location}");
          return sb.ToString();
       }
 
-      private static string BuildPlayerList(Dictionary<SocketGuildUser, int> list)
+      private static string BuildPlayerList(ImmutableDictionary<SocketGuildUser, int> players)
       {
-         if (list.Count == 0)
+         if (players.Count == 0)
             return "-----";
 
          StringBuilder sb = new StringBuilder();
-
-         foreach (KeyValuePair<SocketGuildUser, int> player in list)
+         foreach (var player in players)
          {
             string teamString = GetPlayerTeam(player.Key);
             sb.AppendLine($"{raidEmojis[player.Value - 1]} {player.Key.Nickname ?? player.Key.Username} {teamString}");
          }
+         return sb.ToString();
+      }
+
+      private static string BuildInvitedList(ImmutableDictionary<SocketGuildUser, SocketGuildUser> players)
+      {
+         if (players.Count == 0)
+            return "-----";
 
+         StringBuilder sb = new StringBuilder();
+         foreach (var player in players)
+         {
+            string teamString = GetPlayerTeam(player.Key);
+            sb.AppendLine($"{player.Key.Nickname ?? player.Key.Username} {teamString} invited by {player.Value.Nickname ?? player.Value.Username}");
+         }
          return sb.ToString();
       }
 
       private static string BuildRaidHelpMessage(ulong code)
       {
          StringBuilder sb = new StringBuilder();
-
          sb.AppendLine("Raid Help:");
          sb.AppendLine("The numbers represent the number of accounts that you have with you." +
             " React with one of the numbers to show that you intend to participate in the raid.");
-         sb.AppendLine($"Once you arrive at the raid, react with {raidEmojis[(int)RAID_EMOJI_INDEX.PLAYER_HERE]} to show others that you have arrived." +
-            $" When all players have marked that they have arrived, Nona will send a message to the group.");
+         sb.AppendLine($"Once you are ready for the raid, react with {raidEmojis[(int)RAID_EMOJI_INDEX.PLAYER_READY]} to show others that you are ready." +
```

Comment:
remove ' to jump'

Reply:
removed 'to jump' from here and ping message.